fe/fe-core/src/main/java/com/starrocks/persist/gson/GsonUtils.java

```diff
@@ -336,7 +336,7 @@ public class GsonUtils {
 
     private static final RuntimeTypeAdapterFactory<Warehouse> WAREHOUSE_TYPE_ADAPTER_FACTORY = RuntimeTypeAdapterFactory
             .of(Warehouse.class, "clazz")
-            .registerSubtype(LocalWarehouse.class, "LocalWarehouse");
+            .registerSubtype(DefaultWarehouse.class, "DefaultWarehouse");
```

Comment:
how's the meta compatibility of this change?

Reply:
There is no compatibility issue, there is no serialization code in the warehouse